manifests/application/application-rollout.yaml

```diff
@@ -38,11 +30,10 @@ spec:
     metadata:
       labels:
         app: rollouts-demo
-        istio-injection: enabled
     spec:
       containers:
       - name: rollouts-demo
-        image: argoproj/rollouts-demo:blue
+        image: anaisurlichs/ping-pong:2.0
```

Comment:
2 lines below, we need to change port name from `http` to something like `m-http` so Prometheus can discover it (: